### DISCUSSION TITLE
Handling Calls with WhatsApp Business API

### DISCUSSION THREAD

**Discussion by JimPresting** [gabrieldwight/Whatsapp-Business-Cloud-Api-Net#117] [Q&A]
Hi everyone,

I’ve set up complex RAG systems where the WhatsApp Business API responds in text, audio, or even with images and documents, and it works well. However, I’ve never worked on a system for handling calls to the Business API number. I’d like to build a call center for interacting with knowledge. This would obviously work best with custom GPT models, as RAG systems tend to have too much latency for real-time interaction.

Does anyone have resources or insights on how to connect calls to the GPT-4o Realtime model or potentially ElevenLabs’ latest real-time models (either option works)?

Thanks in advance!

**Comment by gabrieldwight** [comment]
Hi, is business api number related to phone number functions?

**Comment by jayquicksell** [comment]
You can assign specific calls to designated team members, ensuring that the most suitable person addresses each inquiry. This feature helps in streamlining communication and improving response efficiency.

Additionally, the API allows businesses to track the number of calls each team member handles, which aids in monitoring performance and optimizing workload distribution. Businesses can also use the API to schedule calls with clients or consumers.
